### ISSUE TITLE
ERROR inserting into MYSQL

### ISSUE THREAD

**Issue by goharahmed** [sipcapture/homer#174]
Hi,
I just installed sipcapture first time on new Debian 8.4 using shell installer script, pointed my couple of FreeSWITCH servers to send capture traffic to this server. I'm seeing following errors in syslog.
_I've increases debug level in Kamailio to get more_ 

> Jun  3 01:33:20 [20885]: DEBUG: sipcapture [sipcapture.c:1562]: sip_capture(): CAPTURE DEBUG...
> Jun  3 01:33:20 [20885]: DEBUG: sipcapture [sipcapture.c:1859]: sip_capture(): src_ip: [10.132.43.180]
> Jun  3 01:33:20 [20885]: DEBUG: sipcapture [sipcapture.c:1860]: sip_capture(): dst_ip: [10.132.43.231]
> Jun  3 01:33:20 [20885]: DEBUG: sipcapture [sipcapture.c:1862]: sip_capture(): dst_port: [5060]
> Jun  3 01:33:20 [20885]: DEBUG: sipcapture [sipcapture.c:1863]: sip_capture(): src_port: [5080]
> Jun  3 01:33:20 [20885]: DEBUG: sipcapture [sipcapture.c:1507]: sip_capture_store(): insert into homer table [1]: [sip_capture_call_20160603]
> Jun  3 01:33:20 [20885]: DEBUG: sipcapture [sipcapture.c:1510]: sip_capture_store(): storing info...
> Jun  3 01:33:20 [20885]: ERROR: db_mysql [km_dbase.c:128]: db_mysql_submit_query(): driver error on query: Data too long for column 'msg' at row 1 (1406)
> Jun  3 01:33:20 [20885]: ERROR: <core> [db_query.c:235]: db_do_insert_cmd(): error while submitting query
> Jun  3 01:33:20 [20885]: DEBUG: sipcapture [sipcapture.c:1521]: sip_capture_store(): failed to insert into database(first attempt)
> Jun  3 01:33:20 [20885]: ERROR: db_mysql [km_dbase.c:128]: db_mysql_submit_query(): driver error on query: Data too long for column 'msg' at row 1 (1406)
> Jun  3 01:33:20 [20885]: ERROR: <core> [db_query.c:235]: db_do_insert_cmd(): error while submitting query
> Jun  3 01:33:20 [20885]: ERROR: db_mysql [km_dbase.c:128]: db_mysql_submit_query(): driver error on query: Data too long for column 'msg' at row 1 (1406)
> Jun  3 01:33:20 [20885]: ERROR: <core> [db_query.c:235]: db_do_insert_cmd(): error while submitting query
> Jun  3 01:33:20 [20885]: ERROR: db_mysql [km_dbase.c:128]: db_mysql_submit_query(): driver error on query: Data too long for column 'msg' at row 1 (1406)
> Jun  3 01:33:20 [20885]: ERROR: <core> [db_query.c:235]: db_do_insert_cmd(): error while submitting query
> Jun  3 01:33:20 [20885]: ERROR: db_mysql [km_dbase.c:128]: db_mysql_submit_query(): driver error on query: Data too long for column 'msg' at row 1 (1406)
> Jun  3 01:33:20 [20885]: ERROR: <core> [db_query.c:235]: db_do_insert_cmd(): error while submitting query
> Jun  3 01:33:20 [20885]: ERROR: db_mysql [km_dbase.c:128]: db_mysql_submit_query(): driver error on query: Data too long for column 'msg' at row 1 (1406)
> Jun  3 01:33:20 [20885]: ERROR: <core> [db_query.c:235]: db_do_insert_cmd(): error while submitting query
> Jun  3 01:33:20 [20885]: ERROR: db_mysql [km_dbase.c:128]: db_mysql_submit_query(): driver error on query: Data too long for column 'msg' at row 1 (1406)
> Jun  3 01:33:20 [20885]: ERROR: <core> [db_query.c:235]: db_do_insert_cmd(): error while submitting query
> Jun  3 01:33:20 [20885]: ERROR: sipcapture [sipcapture.c:1531]: sip_capture_store(): failed to insert into database(second attempt)
> 

Please advise how to solve this, like modify which table's column to have this fixed !

Regards,
Gohar

**Comment by lmangani** [comment]
Hi Gohar,

Your packets are simply larger than the table column size. You can change
this in the rotation scripts (column msg) which create your data partitions.


Kind Regards,

Lorenzo Mangani
*Managing Director and Core Dev*

QXIP BV - Capture Engineering
Amsterdam, The Netherlands

Desk: +31 20 8080468 / +1 202-632-7227
Mobile: +31 6 4603-2730
Email: info@qxip.net

Find our more about HOMER, PCAPTURE, CAPTAGENT and all our Services and
Solutions at http://qxip.net

CONFIDENTIALITY NOTICE: This e-mail message, including any attachments, is
for the sole use of the intended recipient(s) and may contain confidential or
legally privileged information. Any unauthorized review, use, disclosure or
distribution is prohibited. If you are not the intended recipient,
please contact
the sender by reply e-mail and destroy all copies of this original message.

**Comment by goharahmed** [comment]
Hi Lorenzo,

I've modified this line:

  `msg` varchar(1500) NOT NULL DEFAULT '',
To
  `msg` varchar(2500) NOT NULL DEFAULT '',
Now, I don't think I'm seeing this error - but apparently I've some values in homer_statistics database tables but GUI is not displaying anything . !! No graphs anywhere or traces displayed ?

Thanks,
Gohar

**Comment by adubovikov** [comment]
please check if your api works correct

i.e. http://yourhost/api/test

**Comment by goharahmed** [comment]
Just tried, gave me a blank page.

--
*GOHAR AHMED *


*Sr. VoIP Systems Software Engineer *

*Tel: *+1.519.807.2240 *|* *Skype: *scribedfor

www.saevolgo.ca

<http://www.linkedin.com/pub/gohar-ahmed/35/81b/107>
<https://www.facebook.com/saevolgo/>  <https://twitter.com/saevolgo>
<http://saevolgo.blogspot.com/>
<https://www.packtpub.com/virtualization-and-cloud/implementing-citrix-xenserver-quickstarter>

*- SaevolGo delivers reliable VoIP services - *

On Fri, Jun 3, 2016 at 4:36 PM, Alexandr Dubovikov <notifications@github.com

**Comment by adubovikov** [comment]
https://github.com/sipcapture/homer/wiki/FAQ#q-i-see-errors-related-to-api-calls-404-not-found

or please follow our WIKI

**Comment by lmangani** [comment]
Hi @goharahmed is this still a problem or you solved the API setup issues?

**Comment by goharahmed** [comment]
Hi Lorenzo,
Yeah kindof. Had to redo another round of installations and got things
working.
One thing that I have observed is that when I point any of my Kamailio or
FS and make test calls they dont get displayed in GUI in realtime. !
Anyways I take it as lack of study and focus on my part and I really
appreciate your work and followup on this.
I will definitely reply back with my queries frequently in near future
along with possibly my blog post on how I tackled my initial hurdles
working with Homer.

Thanks again,
Regards,
Gohar
